src/ServiceHub.Room.Service/Controllers/RoomController.cs

```diff
@@ -191,13 +196,15 @@ public RoomController(ILoggerFactory loggerFactory, IRoomsRepository repo) : bas
                     {
                         await _context.UpdateAsync(newCtxItem);
                     }
-                    catch
+                    catch(Exception e)
                     {
+                        logger.LogError(e, "Room model cannot be update a room within the database", null);
                         return StatusCode(500);
                     }
                 }
                 else
                 {
+                    logger.LogError("Room in database trying to be updated cannot be found", null);
```

Comment:
This isn't a database error. Here, the model would somehow be invalid if it was valid on line 190.